assets/tests/components/ladderPage.test.tsx

```diff
@@ -423,9 +427,9 @@ describe("LadderPage", () => {
     ;(useVehicles as jest.Mock).mockImplementationOnce(() => ({
       ["1"]: [vehicle],
     }))
-    ;(useTimepoints as jest.Mock).mockImplementationOnce(
-      () => timepointsByRouteId
-    )
+    ;(useTimepoints as jest.Mock)
+      .mockImplementationOnce(() => timepointsByRouteId)
+      .mockImplementationOnce(() => timepointsByRouteId)
```

Comment:
question: Why use `mockImplementationOnce` twice instead of `mockImplementation`?

Reply:
[This comment](https://github.com/mbta/skate/pull/1822#discussion_r1036064847) _may_ be related - I've previously run into wanting to call `mockImplementationOnce` twice due to multiple rerenders when calling `useSocket` instead of getting the socket from context.

Reply:
I wasn't fully sure about this actually but @KaylaBrady was right - since I'm making the change to use the socket from the context I can go back to just a single call to `mockImplementationOnce`.